easybuild/easyblocks/g/gromacs.py

```diff
@@ -314,7 +314,8 @@ def test_step(self):
             # make very sure OMP_NUM_THREADS is set to 1, to avoid hanging GROMACS regression test
             env.setvar('OMP_NUM_THREADS', '1')
 
-            self.cfg['runtest'] = 'check'
+            # run 'make check' in parallel since it involves more compilation
+            self.cfg['runtest'] = 'check -j %s' % self.cfg['parallel']
```

Comment:
@akesandgren @ocaisa's suggestion is valid here: the `parallel` easyconfig parameter may not be set (at least in theory, it's pretty uncommon), so let's do this instead:

```python
self.cfg['runtest'] = 'check'
if self.cfg['parallel']:
    # run 'make check' in parallel since it involves more compilation
    self.cfg.update('runtest', "-j %s" % self.cfg['parallel'])
```